spanner/api/Spanner/Program.cs

```diff
@@ -2264,14 +2264,14 @@ await cmd.ExecuteReaderAsync().ConfigureAwait(false))
                 // Define create table statement for table with 
                 // commit timestamp column.
                 string createTableStatement =
-               @"CREATE TABLE Performances (
-				SingerId	INT64 NOT NULL,
-				VenueId		INT64 NOT NULL,
-				EventDate	Date,
-				Revenue   INT64,
-				LastUpdateTime TIMESTAMP NOT NULL OPTIONS (allow_commit_timestamp=true)
-			    ) PRIMARY KEY (SingerId, VenueId, EventDate),
-			    INTERLEAVE IN PARENT Singers ON DELETE CASCADE";
+                @"CREATE TABLE Performances (
+                    SingerId       INT64 NOT NULL,
+                    VenueId        INT64 NOT NULL,
+                    EventDate      Date,
+                    Revenue        INT64
```

Comment:
Please add back the trailing comma symbol on this line:

"Revenue        INT64,"

Reply:
Fixed.